HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1607,6 +1827,22 @@ def self.affected_by_vacancy
     return affected_by_vacancy
   end
 
+  def self.affected_by_outage
+    affected_by_outage = {}
+    column_names = SchedulesFile.ColumnNames
+    column_names.each do |column_name|
+      affected_by_outage[column_name] = true
+      next unless ([ColumnOccupants,
+                    ColumnSleeping] +
+                    SchedulesFile.HVACSetpointColumnNames +
+                    SchedulesFile.WaterHeaterColumnNames +
+                    SchedulesFile.BatteryColumnNames).include? column_name
```

Comment:
Compared to vacancy periods, outage periods additionally affect fridge, freezer, pools, hot tubs (and doesn't affect occupants).